### PR TITLE
reports state for it's current functionality

### PR DESCRIPTION
**Pull request by c-Monster**

### CODE REVIEW

File: helpers/status.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	FirmwareVersion string `json:"FirmwareVersion",omitempty`
+	FirmwareVersionAvailable bool `json:"FirmwareVersionAvailable"`
+	SupportSB bool `json:"SupportSB",omitempty`
+	Result bool `json:"Result",omitempty`
+}
+
+func GetCurrentInputs(address string) (map[int]int, error) {
+
+	routerMap := make(map[int]int)
+	const NUM_OUTPUTS = 4
```
Comment (xuther) [inline helpers/status.go]: How hard would it be to get the number from PulseEight. I know they have 8x8 matricies and 8x4 and 4x2 as well.

File: helpers/status.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	HDCP int `json:"HDCP",omitempty`
+	HasSignal bool `json:"HasSignal",omitempty`
+	LinkStatus string `json:"LinkStatus",omitempty`
+	FirmwareVersion string `json:"FirmwareVersion",omitempty`
+	FirmwareVersionAvailable bool `json:"FirmwareVersionAvailable"`
+	SupportSB bool `json:"SupportSB",omitempty`
+	Result bool `json:"Result",omitempty`
+}
+
+func GetCurrentInputs(address string) (map[int]int, error) {
```
Comment (xuther) [inline helpers/status.go]: I might add a 'get current input by output port' endpoint as well. 

That way we can return a JSON response like 
```
{
"input": "{{value}}"
}
```

Which will probably be similar to the non-switcher requests, so we can re-use code in the API when we're compiling responses.

File: helpers/status.go (hunk 1 above)
Comment (xuther) [inline helpers/status.go]: If it's horribly difficult, I wouldn't worry about it for now, but you might want to make it a field we can pass in (and then maybe a separate endpoint for the 4x4 matricies) on the off chance we get a different sized pulse-eight box.

File: helpers/status.go (hunk 2 above)
Comment (c-Monster) [inline helpers/status.go]: seems reasonable to me

File: helpers/status.go (hunk 1 above)
Comment (c-Monster) [inline helpers/status.go]: I'll give it a shot

File: helpers/status.go (hunk 2 above)
Comment (c-Monster) [inline helpers/status.go]: Can we expect the port parameter in the URL to be an int?

Comment (xuther) [inline helpers/status.go]: Yes, we're handle that translation in the DB.

File: helpers/status.go (hunk 1 above)
Comment (xuther) [inline helpers/status.go]: Excellent.

**Pull request merged**